### PR TITLE
[API] service-environments-costs: Return total_costs as null when costs are not accepted

### PR DESCRIPTION
**Pull request by mkurek**
Return null instead of 0 for days (months), where costs were not accepted

### CODE REVIEW

File: src/ralph_scrooge/rest_api/public/v0_10/service_environment_costs.py [hunk 1]
```diff
@@ -378,7 +378,16 @@ def fetch_costs(
...
     for date_ in date_range_:
-        total_cost_for_date = total_costs.get(date_, 0)
+        total_cost_for_date = total_costs.get(
+            date_,
+            # None, when costs for particular date were not accepted (if
+            # accepted costs were requested) otherwise 0, which means, that
+            # costs for particular date were accepted, but this service-env
+            # does not have any costs then
+            # TODO(mkurek): consider accepting only part of the month
+            # (currently only first day of month is checked)
```
Comment (xor-xor) [inline src/ralph_scrooge/rest_api/public/v0_10/service_environment_costs.py]: Could you expand this last comment a little bit by e.g. providing a link to the part of the code that "only first day of the month is checked") refers to..?

File: src/ralph_scrooge/tests/rest_api/public/test_service_environment_costs.py [hunk 2]
```diff
@@ -946,3 +951,141 @@ def test_if_all_costs_are_returned_when_accept_only_param_set_to_false_given(sel
...
+        )
+        # 2016-12-14 is accepted, but sice accepted_only is set to False,
+        # total_cost will be NOT None
+        self.assertIsNotNone(
+            costs['service_environment_costs'][1]['total_cost']
+        )
+
+    @data(False, True)  # `forecast` param
+    def test_if_only_accepted_costs_are_returned_when_accept_only_param_given_and_grouped_by_month(self, forecast):  # noqa: E501
+        months = [
```
Comment (xor-xor) [inline src/ralph_scrooge/tests/rest_api/public/test_service_environment_costs.py:966]: Could you briefly describe what is the final result of this 966-1000 block of code..? (the same applies for that other test here)

**Review by xor-xor — APPROVED** [review]
It's OK for me, only two small remarks re: comments.

**Comment by mkurek** [comment]
@xor-xor take a look now

**Pull request merged**